<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1256</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BoB PC 재고 관리 프로그램</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>                PC방 재고 관리 프로그램</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_View">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>전체 재고 조회</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_enroll">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>재고 등록</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_manage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>재고 관리</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>판매처 정보</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>1061</width>
      <height>571</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Search">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Search">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>76</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>품명</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>분류</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>판매처</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
